src/components/Collapsible/Collapsible.scss

```diff
@@ -1,33 +1,30 @@
 @import '../../styles/common';
 
 .Collapsible {
-  overflow: hidden;
-  max-height: 0;
```

Comment:
Just writing a note to myself to tophat this: https://github.com/Shopify/polaris-react/pull/1980

Reply:
Looks good 😍